### PR TITLE
Fix separator and spacer blocks after api v2 refactoring.

### PR DESCRIPTION
**Pull request by youknowriad**
This also introduces the "View" primitive between mobile and web. The main question for me here is what props this primitive should accept. Implicitly here, it already accepts a bunch of web props (className, style, role, ref)  generated by useBlockProps but I guess the native version of useBlockProps could generate different ones (or we find a device-agnostic way to define these styles and properties).

### CODE REVIEW

File: packages/primitives/src/view/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const View = 'view';
```
Comment (ellatrix) [inline packages/primitives/src/view/index.js:1]: What will this return in the browser?

Comment (youknowriad) [reply packages/primitives/src/view/index.js:1]: bug, it should be div :P

**Review by ellatrix — APPROVED** [review]

File: packages/block-library/src/spacer/edit.js [hunk 2]
```diff
@@ -48,7 +49,7 @@ const SpacerEdit = ( {
 
 	return (
 		<>
-			<div { ...useBlockProps() }>
```
Comment (ellatrix) [inline packages/block-library/src/spacer/edit.js:51]: Ideally we should avoid this and make the wrapper itself resizable. I'll look into that.

Comment (youknowriad) [reply packages/block-library/src/spacer/edit.js:51]: yes, I tried it and I believe there's something with the "ref" not being forwarded properly to the third-party dependency.

File: packages/primitives/src/view/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# View
+
+A drop-in replacement for the div element that works across devices.
```
Comment (Tug) [inline packages/primitives/src/view/README.md:3]: Really like this 😍

**Review by geriux — APPROVED** [review]
LGTM! Tested it on the mobile demo app both iOS and Android, the `Spacer` and `Separator` block work as expected.

Thanks for changing the code to make it work for both web and mobile!

**Pull request merged**